README.md

```diff
@@ -1,3 +1,65 @@
 # Python version
 
 I am using python version 3.11 and above
+
+# Project's Title
+
+FastAPI App
+
+# Description
+
+This is simple web application that allows create and get data from database
+
+# Technologies used
+
+FastAPI - A framework for rapid API development with asynchronous support and Swagger/OpenAPI automatic documentation.
+Python - Programming language
+Uvicorn - ASGI server used to run FastAPI applications.
+Git - A version control system for code collaboration
+HTTP - The project is built on the principles of the HTTP protocol
```

Comment:
I'd better removed python, git and http at all. Moreover, other thechnologies also not needed, because if you add/remove something you will need to change it in 2 places (here in readme and in requirements.txt)